azure-pipelines-docs-integration.yml

```diff
@@ -0,0 +1,109 @@
+trigger:
+  branches:
+    include:
+    - pre_pr_docs-*
+    #- develop
+    #- main
```

Comment:
Keeping this so that they can be manually run for now

Reply:
If develop and main are not needed, why not just delete them?..

Reply:
👍🏼